krampoline/src/pages/SelectPage.tsx

```diff
@@ -33,7 +33,7 @@ const SelectPage = () => {
         sessionStorage.setItem('quizId', quizId);
       }
       if (content) {
-        sessionStorage.setItem('contents', content);
+        sessionStorage.setItem('contents', content.content);
```

Comment:
content 정의를 아래처럼 해주면 content로만 사용할 수 있어요 !
```suggestion
        const { content } = await getQuizContent(quiz)'
        
        sessionStorage.setItem('contents', content);
```

Reply:
넵 객체분할 대신 사용해주셔서 감사합니다 ! 수정해서 다시 올리겠습니다